frontend/src/types/imgixAPITypes.ts

```diff
@@ -6,10 +6,40 @@ export type ImgixGETSourcesData = {
   type: string;
 }[];
 
-export type ImgixGETImagesData = {
+export type ImgixGETAssetsData = {
   attributes: {
-    name: string;
+    analyzed_content_warnings: boolean;
+    analyzed_faces: boolean;
+    analyzed_tags: boolean;
+    categories: null | [];
+    color_model: string;
+    color_profile: string;
+    colors: {};
+    content_type: string;
+    custom_fields: null | string | string[];
+    date_created: number;
+    date_modified: number;
+    description: null | string;
+    dpi_height: number;
+    dpi_width: number;
+    face_count: null | string | number;
+    file_size: number;
+    has_frames: boolean;
+    media_height: number;
+    media_kind: null | string;
+    media_width: number;
+    name: null | string | string[];
+    origin_path: string;
+    source_id: string;
+    tags: {};
+    uploaded_by: null | string | string[];
+    uploaded_by_api: boolean;
+    warning_adult: number;
+    warning_medical: number;
+    warning_racy: number;
+    warning_spoof: number;
+    warning_violence: number;
```

Comment:
We probably won't need all these fields to start. Going to narrow them down to the must haves.

```suggestion
    description: null | string;
    name: null | string;
    origin_path: string;
    source_id: string;
```